### PR TITLE
Xcode 11.4 Fix

### PR DESCRIPTION
**Pull request by mRs-**
This fixes working with Xcode 11.4 and enables Whole Module Optimization for the Targets.

Use ENABLE_TESTING_SEARCH_PATHS = YES

### CODE REVIEW

File: QuickSwiftCheck.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -280,13 +280,16 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 3AD546101F17A0D50000CCCA /* Common-Debug.xcconfig */;
 			buildSettings = {
+				ENABLE_TESTING_SEARCH_PATHS = YES;
```
Comment (akashivskyy) [inline QuickSwiftCheck.xcodeproj/project.pbxproj]: QuickSwiftCheck manages build settings exclusively through xcconfig files. Please remove this setting from here and add it at the end of [Common-Base.xcconfig](https://github.com/akashivskyy/QuickSwiftCheck/blob/develop/QuickSwiftCheck/Configurations/Common/Common-Base.xcconfig):

```none
// MARK: Custom

ENABLE_TESTING_SEARCH_PATHS = YES;
```

File: QuickSwiftCheck.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -280,13 +280,16 @@
...
 			};
 			name = Debug;
 		};
 		3A5222451F13E02A005D985A /* Release */ = {
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 3AD546111F17A0D50000CCCA /* Common-Release.xcconfig */;
 			buildSettings = {
+				ENABLE_TESTING_SEARCH_PATHS = YES;
+				SWIFT_COMPILATION_MODE = wholemodule;
```
Comment (akashivskyy) [inline QuickSwiftCheck.xcodeproj/project.pbxproj]: WMO wasn't enabled for a reason. QuickSwiftCheck is a testing framework. Leaving it unoptimized will maximize debugging capabilities even if tests run in release configuration. Please ignore Xcode's suggestion and remove this setting.

**Review by akashivskyy — CHANGES_REQUESTED** [review]
Thank you so much for your contribution, Marius! 🙌
There's just 2 small things before I can merge this.

**Comment by mRs-** [comment]
Fixed it 👍

**Review by akashivskyy — APPROVED** [review]

**Comment by akashivskyy** [comment]
Thanks! I'll release a new version promptly.

**Comment by akashivskyy** [comment]
And it's live in version 3.1.0. Thanks again!

**Pull request merged**